src/rgw/rgw_request.h

```diff
@@ -56,10 +56,10 @@ struct RGWLoadGenRequest : public RGWRequest {
 	string method;
 	string resource;
 	int content_length;
-	atomic_t* fail_flag;
+	std::atomic<bool>* fail_flag = { nullptr };
```

Comment:
please rebase - these changes were already merged in https://github.com/ceph/ceph/pull/14839